killerbee/dev_apimote.py

```diff
@@ -198,6 +198,9 @@ def pnext(self, timeout=100):
             return None
 
         frame = packet[1:]
+        frame = frame[:-7]
+        frame = frame + "\xff"
```

Comment:
Ok, so I've made a huge mistake. I'm still learning how to use Git. I forked your origin repo to work locally for my personal needs. When I tried to push my modifications to my forked repo, I actually pushed to the origin master. In fact, there is just the commits 8da0492 and cc232cd that are legit. All the others are just errors. I'm working with the ApiMote and I met some problems (extra bytes for each frames received, also trying to increment counter because I saw that my devices always start their association request with a counter>60 ). 
All theses modifications are for my edge case.
Sorry for that.